### ISSUE TITLE
Regarding the working and explanation of upscale loss

### ISSUE THREAD

**Issue by stalin18** [PolyLiYJ/StyleGuard#6]
Hi! I have a conceptual query about the upscale loss in Eq. 4 and the explanation provided in the paper. 

Please refer to the text inside the red brackets in the attached screenshot. It says `Upscale loss function aims to cause purifiers and upscalers to amplify protective perturbations instead of reducing it during the diffusion process`.

Questions:

1. If we see Eq. 4 and Eq. 6, both are maximizing the noise prediction error of a diffusion model, which disrupts the reverse diffusion process. That's how the DreamBooth finetuning by the attacker fails. Similarly, because of Eq. 4, if we apply purifier (e.g., Noise Upscale) to the protected image, the purifier should fail to generate the right image? However, Fig. 7 in the appendix, page 25 shows the applying Noise Upscaling on protected portrait images does not lead to any disruption or wrong output generation. Could you please explain why? What causes this different behavior when the loss function for both DreamBooth and Noise Upscale is same (maximizing the noise prediction at a random step "t").

2. In continuation, what do you mean by `Upscale loss function aims to cause purifiers and upscalers to amplify protective perturbations instead of reducing it during the diffusion process`? How the loss fucntion in Eq. 4 amplify the protective perturbations instead of disrupting the purifier's denoising process?

Thank you in advance for helping out.

![Image](https://github.com/user-attachments/assets/64c5b1f7-cc6c-4ee4-b9ba-5d1c4f1822cd)
